src/lib/invite.js

```diff
@@ -108,6 +100,7 @@ async function claimPointFromInvite({
   web3,
   contracts,
   onUpdate,
+  transferEth,
```

Comment:
let's default this to false (assuming that not transferring eth is the reasonable default)